pkg/oc/cli/cmd/login/loginoptions.go

```diff
@@ -231,10 +232,14 @@ func (o *LoginOptions) gatherAuthInfo() error {
 		// client is connecting to the right host:port
 		if statusErr, ok := err.(*kerrors.StatusError); ok {
 			if statusErr.Status().Code == http.StatusInternalServerError {
-				return fmt.Errorf("error: The server was unable to respond - %v", suggestion)
+				fmt.Fprintf(o.ErrOut, "error: The server was unable to respond - %v", suggestion)
```

Comment:
This needs a newline at the end and you probably want to print the error as well.  Not special casing `StatusInternalServerError` would also be reasonable thing to do.

Reply:
Thanks, will go ahead and remove the special case for internal server err - the error we print is pretty much the same as the one below this